homeassistant/components/honeywell/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Config flow to configure the honeywell integration."""
+import somecomfort
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import (
+    CONF_COOL_AWAY_TEMPERATURE,
+    CONF_DEV_ID,
+    CONF_HEAT_AWAY_TEMPERATURE,
+    CONF_LOC_ID,
+    DEFAULT_COOL_AWAY_TEMPERATURE,
+    DEFAULT_HEAT_AWAY_TEMPERATURE,
+    DOMAIN,
+)
+
+
+class HoneywellConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a honeywell config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Show the setup form to the user."""
+        data_schema = {
+            vol.Required(CONF_USERNAME): str,
+            vol.Required(CONF_PASSWORD): str,
+            vol.Optional(
+                CONF_COOL_AWAY_TEMPERATURE, default=DEFAULT_COOL_AWAY_TEMPERATURE
+            ): vol.Coerce(int),
+            vol.Optional(
+                CONF_HEAT_AWAY_TEMPERATURE, default=DEFAULT_HEAT_AWAY_TEMPERATURE
+            ): vol.Coerce(int),
```

Comment:
Only ask for the bare minimum to get the entry set up.
These things are not needed during set up, and can be moved into an option flow.

Reply:
These 2 settings almost seem like something that should be a service to update, rather than config options.